stock_picking_backorder_strategy/i18n/am.po

```diff
@@ -0,0 +1,72 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# * stock_picking_backorder_strategy
+#
+# Translators:
+msgid ""
+msgstr ""
+"Project-Id-Version: stock-logistics-workflow (8.0)\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2016-09-13 22:02+0000\n"
+"PO-Revision-Date: 2015-10-21 16:33+0000\n"
+"Last-Translator: <>\n"
+"Language-Team: Amharic (http://www.transifex.com/oca/OCA-stock-logistics-"
+"workflow-8-0/language/am/)\n"
+"Language: am\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#. module: stock_picking_backorder_strategy
+#: model:ir.model.fields,field_description:stock_picking_backorder_strategy.field_stock_picking_type_backorder_strategy
```

Comment:
Starting from v12, a field external ID follows the form `field_model_name__field_name`, so in this case it would be:
`field_stock_picking_type__backorder_strategy`.